src/sql/src/plan/statement/dml.rs

```diff
@@ -886,6 +950,35 @@ pub fn plan_copy(
             }
             _ => sql_bail!("COPY FROM {} not supported", target),
         },
+        (CopyDirection::To, CopyTarget::Url(url_expr)) => {
+            scx.require_feature_flag(&vars::ENABLE_COPY_TO_URL)?;
+
+            let from = match relation {
+                CopyRelation::Table { name, columns } => {
+                    if !columns.is_empty() {
+                        // TODO(mouli): Add support for this
+                        sql_bail!("Specifying columns for COPY <table_name> TO commands not yet supported. Please use COPY (SELECT...) TO ... instead");
+                    }
```

Comment:
This was technically not in scope when the design doc was written, but could be a nice to have.